components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -80,6 +80,10 @@ AtmosphereOutput (const ekat::Comm& comm, const ekat::ParameterList& params,
 {
   using vos_t = std::vector<std::string>;
 
+  if (params.isParameter("Fill Value")) {
+    m_fill_value = static_cast<float>(params.get<double>("Fill Value"));
```

Comment:
Commenting here because code I reference isn't in the PR:

For vertical and horizontal remapping we can specify the "mask" value, currently set to std::numeric_limits<float>.  No that we allow for a fill_value can we have the mask value overridden if `m_fill_value` is set?

Reply:
We could. Do you want me to do it in this PR?